### DISCUSSION TITLE
Mage Training Arena

### DISCUSSION THREAD

**Discussion by Shunolog** [runelite/runelite#15354] [Ideas]
First of all, thank you for this wonderful plug in. A small suggestion in telekinetic room, where a tile is marked where to stand and telekinetic the guardian from, if the next standing tile could also be highlighted :)

**Comment by MrFluffums** [comment]
This would improve the flow of the Telekinesis room significantly, as not knowing where the next tile is can waste energy or require the player to stand still and wait until the statue is finished moving, when they could already be standing in their next position with a spell ready to be cast. This adds unnecessary time to an activity that already proves arduous for many players.
